.github/actions/run-integration-tests/action.yml

```diff
@@ -40,7 +40,7 @@ runs:
             # Forward the SIGINT directly to test process but wait for current
             # active jobs since we can only wait for current shell child process.
             echo "IntegrationTestsJob: Notifying the integration tests process about the cancellation"
-            kill -2 $(pidof integration.test) > /dev/null
+            kill -2 $(pidof inspektor-gadget.test) > /dev/null
```

Comment:
Is it possible to set the name? If so, I would suggest to do that to avoid it changing every time we change the folder name.

Reply:
I didn't find a way to directly do it with `go test` without splitting it into compile and run phase. I have updated the PR let me know if it is looking good now.

Reply:
Oh, I thought there was any way to do that more quickly. Mmmm, I'm not sure about this two-step approach. But if we decide to go with this approach, we should remove the binary after running it.

Reply:
Yes, definitely we should remove the binary but I agree it looks bit too much. Other thought I had was to signal using `pgrep -P $(pidof go)` which removes the issue as well. What do you say?

Reply:
I think just renaming the process should be enough. I don't expect that folder to be renamed too often.

Reply:
Cool. I will revert the changes.